core/src/plugins/filed/python/pyfiles/BareosFdPluginLocalFileset.py

```diff
@@ -132,3 +133,34 @@ def start_backup_job(self):
             return bareosfd.bRC_Error
         else:
             return bareosfd.bRC_OK
+
+
+    def get_xattr(self, xattr):
```

Comment:
I guess people use (and subclass) this class, so maybe we shouldn't add test-code here, but maybe have another subclass for the test.

Reply:
I would assume that people should use `BareosFdPluginLocalFilesBaseclass` for that purpose?

Reply:
I hope that's the case, but we're shipping that file as part of bareos-filedaemon-python-plugins-common - and I really hope nobody is using that plugin.
However, we probably shouldn't have testing-only code in there.

Reply:
Makes sense